### PR TITLE
[208274] Amend button to remove RDI in admin

### PR DESCRIPTION
**Pull request by pkujawa**
[AB#208274](https://unicef.visualstudio.com/4e044e8d-bc28-4768-8074-f80ff6e4a0e5/_workitems/edit/208274)

### CODE REVIEW

File: backend/hct_mis_api/apps/registration_data/admin.py [hunk 1]
```diff
@@ -217,28 +207,18 @@ def generate_query_for_all_grievances_tickets(rdi: RegistrationDataImport) -> Q:
...
                     )
-                    rdi_datahub.delete()
                     GrievanceTicket.objects.filter(
                         RegistrationDataImportAdmin.generate_query_for_all_grievances_tickets(rdi)
                     ).filter(business_area=rdi.business_area).delete()
                     rdi.delete()
                     # remove elastic search records linked to individuals
                     business_area_slug = rdi.business_area.slug
-                    remove_elasticsearch_documents_by_matching_ids(
-                        datahub_individuals_ids, get_imported_individual_doc(business_area_slug)
```
Comment (domdinicola) [inline backend/hct_mis_api/apps/registration_data/admin.py:240]: shouldn't it be like "pending_individuals_ids, get_individual_doc(business_area_slug) "?? @pkujawa

Comment (pkujawa) [reply backend/hct_mis_api/apps/registration_data/admin.py:240]: After Datahub Removal, there are no separate objects for `before` and `after` merging RDI. These are just Household/Individuals already after import and merging updates their status to MERGED. So for the MERGED RDI there are no Pending objects anymore.

**Review by patryk-dabrowski — APPROVED** [review]

**Comment by domdinicola** [comment]
@pkujawa part were "remove the check `is_not_used_in_targeting` and list the TP that will be removed in cascade in the warning message."

**Comment by pkujawa** [comment]
@domdinicola There was no such a requirement before :) But no TP will be removed by a cascade, only HouseholdSelections. I could list TPs that the households were a part of and now they are removed from it. But TPs are not removed

**Comment by domdinicola** [comment]
@pkujawa  mmm but TPs need to have an Household Selection, no?

**Comment by pkujawa** [comment]
@domdinicola It's a ManyToMany between TP and Household, through the `HouseholdSelection`. `HouseholdSelection` is a single Household-TP connection. TP can be without any households

**Comment by domdinicola** [comment]
@pkujawa let's just show some counts of HS and how many Households and Individuals will be removed.

File: backend/hct_mis_api/apps/registration_data/admin.py (hunk 1 above)
Comment (domdinicola) [reply backend/hct_mis_api/apps/registration_data/admin.py:240]: @pkujawa also would be good to amend the description to explain what will be happening (to TP, household, individuals and _tickets_)

**Review by domdinicola — APPROVED** [review]

**Review by johniak — APPROVED** [review]

**Pull request merged**